addons/mail/static/src/new/public/discuss_public_welcome_page.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+<t t-name="mail.discuss_public_welcome_page" owl="1">
+    <div class="o-discuss-public-welcome-page h-100 d-flex flex-column justify-content-center align-items-center bg-light" t-attf-class="{{ className }}" t-ref="root">
+        <h1 class="fw-light">
+            <span t-if="callDemo">You've been invited to a meeting!</span>
+            <span t-if="!callDemo">You've been invited to a chat!</span>
+        </h1>
+        <h2 class="m-5" t-esc="messaging.state.companyName"/>
+        <div class="d-flex justify-content-center">
+            <div class="d-flex flex-column justify-content-center">
+                <t t-if="messaging.state.currentGuest">
+                    <label class="o-discuss-public-welcome-page-guest-name-input-label text-center fs-4" t-att-for="guestNameInputUniqueId">What's your name?</label>
+                    <input class="form-control mb-3" type="text" placeholder="Your name" t-att-name="guestNameInputUniqueId" t-att-id="guestNameInputUniqueId" t-model="state.pendingGuestName" t-ref="guestNameInput" t-on-keydown="onKeydownGuestNameInput"/>
```

Comment:
- Unused `t-ref`
- rename `onKeydownGuestNameInput` => `onKeydownInput` (easier to read and good enough with complexity of template)